File: Harvard - CS50/1pset/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // Get height until proper value inputted
    do{
        height = get_int("Height is: ");
    }while(height < 0 || height > 23);

    for (int i = 0; i < height; i++){

        int hash = i + 1;
        int space = height - hash;

        //print out spaces
        for (int j = 0; j < space; j++){
            printf(" ");
        }
        // print left pyramid hashes
        for (int j = 0; j < hash; j++){
            printf("#");
        }
        // print pyramid separator spaces
        for (int j = 0; j < 2; j++){
            printf(" ");
        }
        // print right pyramid hashes
        for (int j = 0; j < hash; j++){
            printf("#");
        }
        printf("\n");
    }
}

File: Harvard - CS50/6pset/sentimental/mario/more/mario.py
from cs50 import get_int

def drawHash(height):
        for k in range(height):
            print("#", end="")

def drawSpace(space):
    for j in range(space):
        print(" ", end="")

def main():
    while True:
        height = get_int("Height is: ")
        if (height >= 0 and height <= 23):
            break
    for i in range(height):
        symbol = i + 1
        space = height - symbol
        drawSpace(space)
        drawHash(symbol)
        print ("  ", end="")
        drawHash(symbol)
        print()

if __name__ == "__main__":
    main()